.htaccess

```diff
@@ -37,6 +37,8 @@
 	RewriteRule ^(.*)/topics/testing/create-silverstripe-test$ /$1/topics/testing/creating-a-silverstripe-test [R=301,L]
 	RewriteCond %{REQUEST_URI} !(3.0|2.4)
 	RewriteRule ^(.*)/topics/testing/create-functional-test$ /$1/topics/testing/creating-a-functional-test [R=301,L]
+	
+        RewriteRule ^changelog/?$ /changelogs [R=301,L] 
```

Comment:
Sorry, Cam, that was a bit of a clumsy
      start on my part! Distracted at the moment.
      
      Hopefully these can be tested now. 
      
      Will do rewrites over weekend.
      
      Cheers,
      
      David.
      
      
      On 09/07/15 16:13, Cam Findlay wrote:
    
    
      In .htaccess:
      > @@ -37,6 +37,8 @@
>  	RewriteRule ^(.*)/topics/testing/create-silverstripe-test$ /$1/topics/testing/creating-a-silverstripe-test [R=301,L]
>  	RewriteCond %{REQUEST_URI} !(3.0|2.4)
>  	RewriteRule ^(.*)/topics/testing/create-functional-test$ /$1/topics/testing/creating-a-functional-test [R=301,L]
> +	
> +        RewriteRule ^changelog/?$ /changelogs [R=301,L] 

      RewriteRule ^changelog/?$ /en/changelogs [R=301,L]
      —
        Reply to this email directly or view
          it on GitHub.
      
        
          
          
        
        
      
    
    
    
    --